wluma.service

```diff
@@ -8,6 +8,7 @@ ExecStart=/usr/bin/wluma
 Restart=always
 EnvironmentFile=-%E/wluma/service.conf
 PrivateNetwork=true
+PrivateMounts=true
```

Comment:
Just to confirm, this should be false then?

Reply:
oops, my bad